style.css

```diff
@@ -198,6 +202,9 @@ ol, ul, li, p {
 li, p {
   margin-left: 0;
 }
+address {
+  font-style: normal;
+}
 
 .github-fork-ribbon-wrapper {
   display: none;
```

Comment:
As per <https://greenbytes.de/tech/webdav/draft-iab-html-rfc-03.html#element.address>, approved by the IAB. That's why I set override it in the CSS.